pkg/cli/model.go

```diff
@@ -53,24 +51,18 @@ func modelSet(cmd *cobra.Command, args []string) error {
 	if err != nil {
 		return err
 	}
-	cwd, err := os.Getwd()
-	if err != nil {
-		return err
-	}
-	exists, err := files.Exists(filepath.Join(cwd, global.ConfigFilename))
+
+	projectDir, err := config.GetProjectDir(projectDirFlag)
 	if err != nil {
 		console.Error(err.Error())
 	}
-	if !exists {
-		console.Warnf("%s does not exist in %s. Are you in the right directory?", global.ConfigFilename, cwd)
-	}
 
 	cli := client.NewClient()
 	if err := cli.CheckRead(model); err != nil {
 		return err
 	}
 
-	userSettings, err := settings.LoadProjectSettings(cwd)
+	userSettings, err := settings.LoadProjectSettings(projectDir)
```

Comment:
Does it make sense to pass in `projectDir` here instead of `cwd`? I'm not 100% clear on what the intent is/was.

Reply:
Yep, I think this was a bug!

Reply:
Oh I definitely knew that and intended to fix it! :roll_eyes: